### ISSUE TITLE
Feedback: Check installation requirements

### ISSUE THREAD

**Issue by evilrobot-01** [r0gue-io/pop-cli#69]
Feedback received: 

> Small nit in the documentation: cargo install doesn't work if you don't have nightly as default (or add +nightly)

It might be especially useful to get this checked via CI so we detect any regressions once resolved.

**Comment by weezy20** [comment]
Add instructions for building from source? 
```md
cargo build --release && cp ./target/release/pop /usr/local/bin
```

**Comment by evilrobot-01** [comment]
Tried installing from `rust` Docker image, required the installation of `cmake` only and then `pop` installed fine. `rust:slim` required `cmake build-essential libssl-dev pkg-config curl` but then also worked fine.

Seems this was resolved by #73 so closing.

**Comment by evilrobot-01** [comment]
Tried installing from `rust` Docker image, required the installation of `cmake` only and then `pop` installed fine. `rust:slim` required `cmake build-essential libssl-dev pkg-config curl` but then also worked fine.

Seems this was resolved by #73 so closing.
